src/Microsoft.DotNet.Kusto/KustoHelpers.cs

```diff
@@ -175,38 +176,55 @@ public class KustoIngestClientFactory : IKustoIngestClientFactory
     private readonly ConcurrentDictionary<string, IKustoIngestClient> _clients = new ConcurrentDictionary<string, IKustoIngestClient>();
     private readonly ConcurrentDictionary<string, ICslAdminProvider> _adminClients = new ConcurrentDictionary<string, ICslAdminProvider>();
 
+    private string KustoClusterUri => _kustoOptions.CurrentValue.KustoClusterUri;
+    private string KustoIngestionUri => _kustoOptions.CurrentValue.KustoIngestionUri;
+    private string DatabaseName => _kustoOptions.CurrentValue.Database;
+    private string ManagedIdentityId => _kustoOptions.CurrentValue.ManagedIdentityId;
+    private bool UseAzCliAuthentication => _options.CurrentValue.UseAzCliAuthentication;
+
     public KustoIngestClientFactory(IOptionsMonitor<KustoOptions> options)
     {
         _kustoOptions = options;
     }
 
     public IKustoIngestClient GetClient()
     {
-        string ingestConnectionString = _kustoOptions.CurrentValue.IngestConnectionString;
-
-        if (string.IsNullOrWhiteSpace(ingestConnectionString))
-            throw new InvalidOperationException($"Kusto {nameof(_kustoOptions.CurrentValue.IngestConnectionString)} is not configured in settings or related KeyVault");
+        if (string.IsNullOrWhiteSpace(KustoIngestionUri))
+            throw new InvalidOperationException($"Kusto {nameof(_kustoOptions.CurrentValue.KustoIngestionUri)} is not configured in settings");
```

Comment:
nit: could we use `nameof(KustoOptions.KustoClusterUri)`, `nameof(KustoOptions.KustoIngestionUri)`, and similar consistently❔

Reply:
my bad, fixed